resources/views/store/orders/index.blade.php

```diff
@@ -46,24 +46,26 @@
                         @endforeach
                     </ul>
                     @if ($order->isShopify())
-                        <button
-                            class="js-store-resume-checkout btn-osu-big btn-osu-big--rounded-thin"
-                            data-order-id="{{ $order->getKey() }}"
-                            data-provider="{{ $order->getPaymentProvider() }}"
-                            data-provider-reference="{{ $order->getProviderReference() }}"
-                            data-status="{{ $order->status }}"
-                        >
-                            {{ $order->isPaymentRequested() ? osu_trans('store.order.resume') : osu_trans('store.order.invoice') }}
-                        </button>
+                        @if ($order->shopify_url !== null)
+                            <a class="btn-osu-big btn-osu-big--rounded-thin" href="{{ $order->getShopifyUrl() }}">
```

Comment:
point to invoice page instead and have the action redirect to the keyed url for shopify orders?